webui/react/src/components/UPlot/UPlotChart.tsx

```diff
@@ -219,18 +164,22 @@ const UPlotChart: React.FC<Props> = ({
         });
       }
     } else {
-      const isZoomed: boolean = Object.values(boundsRef.current).some((i) => i.isZoomed === true);
-      const propsProvideScales: boolean = Object.values(extendedOptions?.scales ?? {}).some(
-        (scale) => scale.min != null || scale.max != null,
-      );
-      const onlyOneXValue: boolean = Array.isArray(data) && data[0]?.length === 1;
-      const customScaleIsInEffect = isZoomed || (propsProvideScales && onlyOneXValue);
-      const resetScales = !customScaleIsInEffect;
-
       try {
         if (chartRef.current && isReady){
-          chartRef.current.setData(data as AlignedData, resetScales);
-          if (onlyOneXValue) chartRef.current.redraw(true, false);
+          const disp =
+          title === 'Throughput'
+            ? data.map((ser) => ser.slice(0, Math.floor(ser.length / 2)))
```

Comment:
This is for testing what happens if one chart has very different extent of data than the others